src/client/api/pythonApi.ts

```diff
@@ -266,7 +266,7 @@ export class EnvironmentActivationService implements IEnvironmentActivationServi
     ): Promise<NodeJS.ProcessEnv | undefined> {
         return this.apiProvider
             .getApi()
-            .then((api) => api.getActivatedEnvironmentVariables(resource, interpreter, true));
+            .then((api) => api.getActivatedEnvironmentVariables(resource, interpreter, false));
```

Comment:
I think this is incorrect, if we do'nt have environment variables, we can still try to run the kernel.
Python extension has logging to capture the failure to get env vars & the necessary telemetry as well.

The rationale is, no point not launching the kernel because we couldn't get the activated environment variables, lets at least try to launch the kernel without that, and it might work in a majority of the case.
& if we have issues we can track the error logs and see that it has failed.